lettre/readme.txt

```diff
@@ -0,0 +1,76 @@
+=== Lettre ===
+Contributors: Automattic
+Requires at least: 5.8
```

Comment:
update the min version also to 6.1 as it implements fluid typography

Reply:
Oh yeah!